.circleci/config.yml

```diff
@@ -249,6 +292,11 @@ workflows:
       - integration_test:
           requires:
             - build
+      - accessibility_test:
+          requires:
+            - build
+          context:
```

Comment:
no need for the context unless you're going to report (back to slack) upon failure

Reply:
Removed